### PR TITLE
Sprint12 05 a46709 document block device resource

### PR DESCRIPTION
**Pull request by arangamani**
The resources in the block_device cookbook are documented by taking db/resources/default.rb  as an example.

### CODE REVIEW

**Comment by douglaswth** [comment]
In the interest of getting this documented this looks fine. However, I think there will be some changes needed when we start using the YARD Chef tool, so we'll probably revisit this then. Go ahead and merge.

**Comment by arangamani** [comment]
Merging it now.

File: cookbooks/block_device/resources/default.rb [hunk 1]
```diff
@@ -18,31 +30,32 @@
...
 attribute :cron_backup_hour, :kind_of => String
 attribute :cron_backup_recipe, :kind_of => String, :default => "block_device::do_backup"
-# Rotation options
+
+
+# == Rotation options
 attribute :max_snapshots, :kind_of => String
 attribute :keep_daily, :kind_of => String
-attribute :keep_weekly, :kind_of => String
+sattribute :keep_weekly, :kind_of => String
```
Comment (lopaka) [inline cookbooks/block_device/resources/default.rb:47]: Doug...you missed this :)

Comment (arangamani) [inline cookbooks/block_device/resources/default.rb:47]: My bad :(

**Pull request merged**